packages/amqp-transport/package.json

```diff
@@ -23,8 +23,8 @@
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@nestjs/common": "10",
-    "@nestjs/microservices": "10",
+    "@nestjs/common": "^10 || ^11",
```

Comment:
I doubt the new server and client implementation can work with NestJS 10, ensure whether this is the case or prepare for a major version bump